Sources/MongoSwift/ChangeStreamDocument.swift

```diff
@@ -0,0 +1,67 @@
+/// An `UpdateDescription` containing fields that will be present in
+/// the change stream document for operations of type `update`.
+public struct UpdateDescription: Codable {
+   /// A document containing key:value pairs of names of the fields
+   /// that were changed, and the new value for those fields.
+   public let updatedFields: Document
+
+   /// An array of field names that were removed from the document.
+   public let removedFields: [String]
+}
+
+/// An enum representing the type of operation for this change.
+public enum OperationType: String, Codable {
+   case insert
```

Comment:
The linter will probably complain that these need to be documented. The docstrings can be short and basically just link to official docs (if relevant ones exist)